src/TableV8/styled.tsx

```diff
@@ -213,12 +214,14 @@ export const TableCell = styled(TableCellComp, {
 `;
 
 // import this to add custom styles to a table cell
-const TableCellWrapperComp = (props: React.PropsWithChildren<{ className?: string }>) => (
-  <div {...props} className={clsx(TABLE_CLASSES.TD_WRAPPER, props.className)} />
-);
+const TableCellWrapperComp = (
+  props: React.PropsWithChildren<{ className?: string; selected?: boolean }>,
+) => <div {...props} className={clsx(TABLE_CLASSES.TD_WRAPPER, props.className)} />;
 export const TableCellWrapper = styled(TableCellWrapperComp, {
   shouldForwardProp: (prop) => isPropValid(prop),
-})``;
+})`
+  background: ${(props) => (props.selected ? COLORS.BACKGROUND_SELECTED : 'transparent')};
```

Comment:
if you're running this locally, the selected row cells will have some padding (white) around them. the styled.tsx file gets refactored in the next PR.